packages/sw-precaching/test/browser-unit/cache-testing.js

```diff
@@ -306,13 +306,61 @@ describe('sw-precaching Test Revisioned Caching', function() {
     });
   });
 
-  it('should cache unrevisioned opaque responses by default', function() {
+  it('should not cache unrevisioned opaque responses by default', function() {
     return window.goog.swUtils.activateSW('data/response-types/opaque-unrevisioned-sw.js')
     .then(() => {
-      throw new Error('Expected SW to fail installation due to caching 404 entry.');
+      throw new Error('Expected SW to fail installation due to caching opaque entry.');
     }, (err) => {
       // NOOP - The error is not to do with the error throw in SW, so nothing
       // to check.
     });
   });
+
+  it('should manage redirected revisioned requests', function() {
+    return window.goog.swUtils.activateSW('data/response-types/redirect-revisioned-sw.js')
+    .then((iframe) => {
+      goog.__TEST_DATA['redirect'].map((redirectPath) => {
```

Comment:
It looks like you're missing a `return`, since there isn't an implicit `return` when an arrow function has a block as its body.

Assuming that this wasn't intentional, could you check other tests that might also follow a similar pattern and add `returns` there as well? I see that line 346 in this file, at least, also seems to be missing it.